### PR TITLE
Main content

### PR DESCRIPTION
**Pull request by irisxu02**

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -39,14 +39,11 @@ <h1>Iris' Portfolio</h1>
     <!-- Footer -->
     <footer class="footer container">
       <h2 class="actiontext">Connect</h2>
-      <div class="navlinks">
-        <!-- TODO: Add social links -->
-        <ul>
+        <ul class="navlinks">
           <li><a href="https://github.com/irisxu02">GitHub</a></li>
           <li><a href="https://www.linkedin.com/in/irisxu02/">LinkedIn</a></li>
           <li><a href="mailto:someone@irisxu02@gmail.com">Email</a></li>
```
Comment (kuanyiwu) [inline portfolio/src/main/webapp/index.html]: mailto:irisxu02@gmail.com

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -22,19 +22,26 @@ <h1><a href="index.html" class="header-text">Iris Xu</a></h1>
...
+    <main class="container">
+      <!-- TODO: About section -->
+      <section id="hero">
+        <h1>Hello, my name is Iris.</h1>
+      </section>
+
+      <div id="content">
+        <h1>Iris' Portfolio</h1>
+        <p>This is my portfolio.</p>
+        <p>Click <a href="/hello">here</a> to see content created by server code.</p>
```
Comment (kuanyiwu) [inline portfolio/src/main/webapp/index.html]: This week we're focusing on client side development and next week will be server-side. Are you adding some server code for this week?

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -90,6 +90,24 @@ body {
...
+
+.main.container {
+  padding-left: 1rem;
+  padding-right: 1rem;
+}
+
+#hero {
+  padding-top: 6rem;
+  padding-bottom: 6rem;
+  animation: 1.4s down-enter cubic-bezier(.25,1,.5,1) backwards;
```
Comment (kuanyiwu) [inline portfolio/src/main/webapp/style.css:167]: This seems to be missing at least the animation-name property. Using the short-hand version for animation will need to follow a certain order. Alternatively you can specify each attribute separately, e.g. `animation-duration: 1.4s`

https://www.w3schools.com/cssref/css3_pr_animation.asp

File: portfolio/src/main/webapp/style.css [hunk 4]
```diff
@@ -88,8 +100,8 @@ body {
 
 .main {
   margin: auto auto;
-  width: 650px;
-  top: 120px;
+  width: 80ch;
```
Comment (kuanyiwu) [inline portfolio/src/main/webapp/style.css]: width is specified as 100% below, you can remove this width.

**Comment by kuanyiwu** [comment]
Can you deploy your latest commits to your appspot.com site so I can view the new changes? Thanks.

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (irisxu02) [reply portfolio/src/main/webapp/index.html]: This part is being deleted.

File: portfolio/src/main/webapp/style.css (hunk 4 above)
Comment (irisxu02) [reply portfolio/src/main/webapp/style.css]: If I change this width it changes the appearance.

**Comment by irisxu02** [comment]
I deployed it just now.

**Pull request merged**